docs/src/commands/build.md

```diff
@@ -47,20 +47,26 @@ The exact meaning of the profile flags may evolve as the platform matures.
 
 ## Target
 
-The `build` command accepts a `--target` argument. This will customize the output files
-to align with a particular type of JS module. This allows wasm-pack to generate either
-ES6 modules or CommonJS modules for use in browser and in NodeJS. Defaults to `browser`.
-The options are:
+The `build` command accepts a `--target` argument. This will customize the JS
+that is emitted and how the WebAssembly files are instantiated and loaded. For
+more documentation on the various strategies here, see the [documentation on
+deployment][deploy].
```

Comment:
i totally see how "deployment" can work as a word to describe what we mean here but it *may* be a bit overloaded for a general programmer reading the docs, which might cause some confusion (i.e. deployment = pushing to a prod server of some sort). maybe we can say usage instead? i think that may be less intimidating.

Reply:
Sure, I've tweaked the wording